packages/components/inputs/localized-multiline-text-input/src/translation-input.styles.ts

```diff
@@ -9,7 +9,12 @@ import {
 // * a disabled-field currently does not display warning/error-states so it takes precedence
 // * a readonly-field cannot be changed, but it might be relevant for validation, so error and warning are checked first
 // how you can interact with the field is controlled separately by the props, this only influences visuals
-const getTextareaStyles = (props) => {
+
+type TTranslateInputStylesProps = {
```

Comment:
Could we use the name based on this filename? 
I was expecting it to be `TTranslationInputStylesProps`